code/modules/admin/verbs/randomverbs.dm

```diff
@@ -543,7 +543,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	var/msg = "<span class='danger'>Admin [key_name_admin(usr)] healed / revived [ADMIN_LOOKUPFLW(M)]!</span>"
 	message_admins(msg)
 	admin_ticket_log(M, msg)
-	SSblackbox.record_feedback("tally", "admin_verb", 1, "Rejuvinate") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+	SSblackbox.record_feedback("tally", "admin_verb", 1, "Rejuvenate") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!

```

Comment:
won't this stat now have 2 entries, one for the old and one for the new?

Reply:
not sure what you mean, this is just to fix a typo

Reply:
This typo fix has consequences on the database, unfortunately. It will have to be done in its own PR, with DB tweaking.
Just revert it and the rest is fine.

Reply:
Yes you can't change any of the ssblackbox details that aren't just the data without breaking statbus